<?xml version="1.0" encoding="UTF-8" ?>

<launch>
    <node pkg="rtei_calib" type="rtei_calib_learn" name="rtei_calib_learn_node" output="screen">
        <param name="config_filename"
               value="$(find rtei_calib)/config/rtei-config-event.yaml"
               type="string"/>
        <param name="shot_screen_save_dir" value="$(find rtei_calib)/scene" type="string"/>
    </node>
</launch>